app/javascript/dashboard/routes/dashboard/settings/reports/components/WootReports.vue

```diff
@@ -167,6 +169,30 @@ export default {
         DESC: this.$t(`REPORT.METRICS.${key}.DESC`),
       }));
     },
+    calculateTrend() {
+      return metric_key => {
+        if (this.accountSummary.previous[metric_key] === 0) {
+          return 0;
+        }
+        return Math.round(
+          ((this.accountSummary[metric_key] -
+            this.accountSummary.previous[metric_key]) /
+            this.accountSummary.previous[metric_key]) *
+            100
+        );
+      };
+    },
```

Comment:
This logic seems to be repeated, can you please create a mixin?

Reply:
Done